polaris-react/src/tokens/token-groups/keyframes.json

```diff
@@ -0,0 +1,7 @@
+{
+  "keyframes-bounce": "{ from, 65%, 85% { transform: scale(1) } 75% { transform: scale(0.85) } 82.5% { transform: scale(1.05) } }",
+  "keyframes-fade-in": "{ to { opacity: 1 } }",
+  "keyframes-fill-up": "{ 0% { width: 0 } }",
```

Comment:
I would like to remove the `keyframes-fill-up` keyframe. Animating width is low performance in CSS and scaling this approach is not good. I think we should revisit removing this keyframe in another way.

Reply:
@alex-page Jammed with @aaronccasanova and @aveline on [removing `keyframes-fill-up`](https://github.com/Shopify/polaris/pull/5427/files#diff-0cc9612fa4d93196670f24484155e7d04a551ead7f4704ea3ac2a23d605cd094). The solution does add quite a few more lines of code to the .tsx file but I'm not sure we could avoid it without using an animation/keyframes. 

Motion still behaves the same:

https://user-images.githubusercontent.com/21976492/162332047-8d25bd8c-b63e-4ccf-8383-9569abc4c8a2.mp4

https://user-images.githubusercontent.com/21976492/162332062-c2290e04-4b0f-416a-b848-bde90a0137eb.mp4

Reply:
I am happy to leave the keyframes as is if it is a worse solution. I do wonder if we could scale the bar instead of using width. e.g. have a bar that fills the container scaleX(1) that fills the whole bar, then `scaleX(0.1)` and move through the transition.

I would be happy to ship this though and do a follow up. Not blocking.

Reply:
Further [simplified the implementation](https://github.com/Shopify/polaris/pull/5427/commits/1a75cb18e09536f12c270a4ce79ca836a6a39984). Basically removed the extraneous `.Animated/.hasAppearAnimation` class and updated to use a single conditional.